### ISSUE TITLE
Compose Desktop won't display UI and logs no exceptions

### ISSUE THREAD

**Issue by ScottPierce** [JetBrains/compose-multiplatform#2049]
If I'm using Kotlin 1.6.10, and Compose `1.0.1`, and then I want to integrate with a compose multiplatform library like https://github.com/Syer10/compose-material-dialogs#date-and-time-picker , but it uses compose `1.1.1`. I add the dependency and all of a sudden my app just won't run. It will still run in the backround, but compose itself shows no errors, or anything. It just stops working.

So I try to fix this by updating my compose version to `1.1.1` but it still doesn't work and silently fails because I need to update to the latest version of https://github.com/arkivanov/Decompose , so I do the update, but it still doesn't work without an error because Decompose only supports compose `1.1.0`.

This was a rabbit hole, and I only have compose and 2 compose libraries working together right now. This can easily balloon out of control.

I encountered similar issues while adding obfuscation to my code base. My app would start, but the UI wouldn't display. It was basically running in headless mode. There should have been some sort of exception logged or something.

**Comment by arkivanov** [comment]
As far as I understand, Compose 1.1.1 should be backward compatible with 1.1.0, so using different libs based on 1.1.1 and 1.1.0 should not be a problem.

Is there anything in the logs?

**Comment by ScottPierce** [comment]
Nothing in the logs.

@arkivanov I made the same initial assumption at first. From what I have seen though 1.1.1 is not the same as 1.1.0. Had to downgrade to 1.1.0 to get everything working with decompose.

**Comment by arkivanov** [comment]
This sounds like something that should not happen. Are there any specific errors in logs?

**Comment by ScottPierce** [comment]
> This sounds like something that should not happen. Are there any specific errors in logs?

This is why I filed the bug :P

I tried to reproduce it exactly like I did yesterday, and now magically everything is working. Compose 1.1.1, Decompose 0.6.0, and material dialogs 0.7.0.

I've seen this multiple times in other circumstances where the UI won't show and there are just no exceptions or errors in the logs, however the app runs perfectly effectively in a headless mode.
